ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs

```diff
@@ -121,8 +121,28 @@ private RCTDeviceEventEmitter EventEmitter
                 }
                 else if ((formData = data.Value<JArray>("formData")) != null)
                 {
-                    // TODO: (#388) Add support for form data.
-                    throw new NotImplementedException("HTTP handling for FormData not yet implemented.");
+                    if (headerData.ContentType == null)
+                    {
+                        headerData.ContentType = "multipart/form-data";
+                    }
+
+                    var formDataContent = new HttpMultipartFormDataContent();
+                    foreach (var content in formData)
+                    {
+                        var fieldName = content.Value<string>("fieldName");
+
+                        var formDataHeaders = content.Value<JArray>("headers");
```

Comment:
`fieldname` is added as part of the multipart form body `HttpMultipartFormDataContent`